### PR TITLE
Move webroot from project.json to Microsoft.AspNet.Hosting.json.

### PR DESCRIPTION
**Pull request by Tratcher**
#257 @davidfowl @BrennanConroy @rustd @phenning

### CODE REVIEW

File: src/BaseTemplates/EmptyWeb/Microsoft.AspNet.Hosting.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "Hosting:WebRoot": "wwwroot"
```
Comment (davidfowl) [inline src/BaseTemplates/EmptyWeb/Microsoft.AspNet.Hosting.json]: Change the casing to be "hosting:webRoot":

Comment (Tratcher) [inline src/BaseTemplates/EmptyWeb/Microsoft.AspNet.Hosting.json]: how about just "hosting:webroot"?

Comment (davidfowl) [inline src/BaseTemplates/EmptyWeb/Microsoft.AspNet.Hosting.json]: better

**Comment by Tratcher** [comment]
Updated

**Comment by phenning** [comment]
Also needs to be removed in the project.json files under src\Rules.

**Comment by Tratcher** [comment]
Updated @phenning

**Comment by Tratcher** [comment]
Or @phenning, do I not need to add Microsoft.AspNet.Hosting.json to all of the Rules projects because it's not different from the base project?

**Comment by davidfowl** [comment]
:shipit: We should just put the server name in here too :smile:

**Comment by phenning** [comment]
@Tratcher You only need to add it to the base templates if the content is identical between ind auth/no auth, etc.

**Comment by Tratcher** [comment]
@davidfowl That would make it harder to have multiple server commands.

**Pull request merged**